exonum/src/events/mod.rs

```diff
@@ -45,7 +45,39 @@ pub type SyncSender<T> = Wait<Sender<T>>;
 /// This kind of events is used to schedule execution in next event-loop ticks
 /// Usable to make flat logic and remove recursions.
 #[derive(Debug, PartialEq)]
-pub enum InternalEvent {
+pub struct InternalEvent(pub(crate) InternalEventInner);
+
+impl InternalEvent {
+    // FIXME: review visibility
```

Comment:
Is this going to be done within the current PR?

Reply:
Yeah. Since `InternalEvent` is hidden from the docs + these methods are used in consensus tests, I plan to leave the methods public and remove `#[doc(hidden)]` (since it's obsolete).